hack/make-rules/test-cmd-util.sh

```diff
@@ -1608,7 +1608,7 @@ run_non_native_resource_tests() {
   kubectl "${kube_flags[@]}" delete foos test --cascade=true
 
   # Make sure it's gone
-  kube::test::get_object_assert foos "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::wait_object_assert foos "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
> the GC has to remove a finalizer from the deletion target after processing dependents one way or another

It's not true. If the policy is "[Background](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L372)", which is the same as `orphanDependents=false`, the object is deleted immediately, no finalizer involved.